src/app/static/src/app/components/plots/choropleth/Choropleth.vue

```diff
@@ -146,7 +156,7 @@ const getStyle = (feature: Feature) => {
 }
 
 const emptyFeature = computed(() => {
-    return selectedAreaIds.value.length == 0
+    return plotData.value.length === 0
```

Comment:
I think this change has broken the boundary changes. You can recreate if you
1. Open the plot, set the detail level to something lower than country e.g. district
2. Update the area to exclude Northern -> no boundary update
3. Update the areas to include Northern -> boundary update to box containing just central and southern districts

If this is "selectedAreaIds" then should update correctly

Reply:
this is a big problem actually, have been messing around with it, this should really be based off `plotData` and not `selectedAreaIds` because:
* currently if you select area level as "Country" and selected area as "Northern" there is no empty map feature which is not the behaviour on production
* since ART data (at least for `MWI_demo`) is only for "District + Metro" area level, there will be `selectedAreaIds` when you are on area level "Country" but there is no plot data so you again dont see an empty map feature but have an empty map

I think the issue is because `plotData` is being fetched so is an async action, but so is the debounced `updateBounds` theres 2 levels of async here and something is basically going wrong with the debounced function, at least that is my working theory

I have reverted this to `selectedAreaIds` for now but we have broken empty map cases we should fix!

Reply:
Good spot, thanks, I've made a ticket here https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-5427/Fix-issues-with-no-empty-data-message-showing